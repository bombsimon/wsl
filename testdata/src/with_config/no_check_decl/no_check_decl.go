package testpkg

func fn1() {
	var a = 1
	var b = 2
	const c = 3
	const d = 4

	e := 5
	var f = 6
	g := 7

	_ = a
	_ = b
	_ = c
	_ = d
	_ = e
	_ = f
	_ = g
}
