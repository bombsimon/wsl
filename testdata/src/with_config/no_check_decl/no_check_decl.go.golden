package testpkg

func fn1() {
	var a = 1
	var b = 2
	const c = 3
	const d = 4
}

